test/averages-test.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('chai').assert
+const Average = require('../lib/averages')
+
+describe('average functionality', function() {
+  context('total(array)', function() {
+    it('returns the total of all values in the array', function() {
+      array = [1, 2, 3]
```

Comment:
I would treat variable the same in the tests and still but `var` before it. Otherwise you could run into problems with using the same variable between multiple tests